exercises/practice/dnd-character/.meta/example.rb

```diff
@@ -0,0 +1,53 @@
+module Die
+  extend self
+
+  def roll(roll: 4, die: 6, remove_lowest: 1)
+    roll.times.map { rand(1..die) }.sort.pop(roll - remove_lowest).sum
+  end
+
+end
+
+module Modifiable
+
+  # This only calculates hit point modifier based on constitution
+  def modifier(constitution)
```

Comment:
Let's ask for the "generic" thing.

Reply:
Using `ability_score` as the new argument based of 5e naming conventions: https://roll20.net/compendium/dnd5e/Ability%20Scores#content